powerdns/tests/test_ownership.py

```diff
@@ -90,3 +101,121 @@ def test_explicit_user(self):
             },
         )
         self.assertOwner(request, 'user2', mailed=True)
+
+
+class TestCreateRecordAccessByServiceOwnership(TestCase):
+
+    def setUp(self):
+        self.clicker = UserFactory(username='clicker')
+        self.some_dude = UserFactory(username='some_dude')
+        self.example_domain = DomainFactory(
+            owner=self.clicker,
+            name='example.com',
+            unrestricted=False,
+            auto_ptr=AutoPtrOptions.NEVER,
+        )
+
+    def _test_create(self, domain_owner, domain_ownership, expected):
```

Comment:
There is a plenty of that here https://github.com/allegro/django-powerdns-dnssec/blob/develop/dnsaas/api/v2/tests.py#L96
Could you tell what tests do you mean in details, so that I could check if they exists or not?